### ISSUE TITLE
Feature request: Tidal information

### ISSUE THREAD

**Issue by vanosg** [n0bel/PiClock#165]
@n0bel this is an amazing project, you did a fantastic job developing it, and an even better job documenting it! Hats off for that.

Do you think there is any chance of integrating tidal information onto a screen? ie, high tide/low tide? If you could rough in a place on the GUI, I'd be happy to code up something to parse an API. NOAA offers an API that might be of use via https://tidesandcurrents.noaa.gov/api/ as an example. Please let me know if I can assist! Thanks again

**Comment by togatown** [comment]
@vanosg what kind of data are you looking for? [Just High/Low times?](https://tidesandcurrents.noaa.gov/api/datagetter?units=english&format=json&station=8721604&date=today&time_zone=lst_ldt&interval=hilo&product=predictions&datum=MLLW)

**Comment by vanosg** [comment]
Yes, just the next high and low (or maybe two sets of high and low, if there is space for it, but I think that may be ambitious)

**Comment by skipwill** [comment]
I added another hand to the PiClock to track the tide movement. 12 o’clock is high tide and 6 o’clock is low tide. The hand moves as it tracks the tide.

**Comment by vanosg** [comment]
This sounds amazing! Is the code available somewhere?

**Comment by skipwill** [comment]
Let me get the openweather conversion done first and I’ll post the tide clock hand code. I am having to convert to qt5 and python 3 as qt4 seems to not be available anymore.

**Comment by n0bel** [comment]
Never integrated, and it won't be here. But it does exist: @skipwill added a
tide hand to the analog clock on top of SerBrynden's fork and posted the code
over in #251 — driven off a NOAA tide station id in the config. That's the best
route if you still want it. Closing this one.
